patches/0027-ipq807x-add-the-Qualcomm-AX-target-support.patch

```diff
@@ -7324,8 +7324,8 @@ index 0000000000..3a219e6d6c
 +  ROOTFSNAME_IN_UBI := ubi_rootfs
 +  BLOCKSIZE := 128k
 +  PAGESIZE := 2048
-+  IMAGES := sysupgrade.tar nand-factory.bin
-+  IMAGE/sysupgrade.tar := sysupgrade-tar | append-metadata
++  IMAGES := sysupgrade.bin nand-factory.bin
++  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
```

Comment:
Is this just renaming the file?
What will happen when sysupgrade is called and the content is in a different format?